From ed3551870f7ec2a3a6245237f48124198e5c9e38 Mon Sep 17 00:00:00 2001
From: Anna-Maria Behnsen <anna-maria@linutronix.de>
Date: Wed, 17 Apr 2024 16:02:29 +0200
Subject: [PATCH] timerqueue: Remove never used function
 timerqueue_node_expires()

References: bsc#1232534 (PREEMPT_RT prerequisite backports)
Patch-mainline: v6.10-rc1
Git-commit: e84c60032a39e3267f0b46175d5368da33e214a6

This function was introduced with commit 60bda037f1dd ("posix-cpu-timers:
Utilize timerqueue for storage") but never used. Remove it.

Signed-off-by: Anna-Maria Behnsen <anna-maria@linutronix.de>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Link: https://lore.kernel.org/r/20240417140229.19633-1-anna-maria@linutronix.de
Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 include/linux/timerqueue.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/include/linux/timerqueue.h b/include/linux/timerqueue.h
index adc80e29168e..ddf02c1fa022 100644
--- a/include/linux/timerqueue.h
+++ b/include/linux/timerqueue.h
@@ -48,11 +48,6 @@ static inline bool timerqueue_node_queued(struct timerqueue_node *node)
 	return !RB_EMPTY_NODE(&node->node);
 }
 
-static inline bool timerqueue_node_expires(struct timerqueue_node *node)
-{
-	return node->expires;
-}
-
 static inline void timerqueue_init_head(struct timerqueue_head *head)
 {
 	head->rb_root = RB_ROOT_CACHED;
